test/test_von_mises.cpp

```diff
@@ -363,7 +387,7 @@ BOOST_AUTO_TEST_CASE( test_main )
       "to pass.</note>" << std::endl;
 #endif
 
-   
+
```

Comment:
Could you add a multiprecision test? Template it on float128 should be sufficient.